treadmill/admin.py

```diff
@@ -391,8 +409,9 @@ def connect(self):
                 server = ldap3.Server(uri)
                 self.ldap = ldap3.Connection(
                     server,
-                    authentication=ldap3.SASL,
-                    sasl_mechanism='GSSAPI',
+                    authentication=ldap3.SIMPLE,
```

Comment:
reading authentication (sasl,simple) from config file to decide the authentication scheme.